### PR TITLE
Use HudUtility list for GQL ProjectType enum

### PR DESCRIPTION
**Pull request by gigxz**
Stop using a manual enum, so we dont have to maintain it in multiple places, and so we use consistent naming with hud.

### CODE REVIEW

**Comment by gigxz** [comment]
should be re-targeted to 86

File: drivers/hmis/app/graphql/types/hmis_schema/enums/project_type.rb [hunk 1]
```diff
@@ -11,19 +11,11 @@ class HmisSchema::Enums::ProjectType < Types::BaseEnum
...
-    value 'SERVICES_ONLY', 'Services Only', value: 6
-    value 'OTHER', 'Other', value: 7
-    value 'SH', 'Safe Haven', value: 8
-    value 'OPH', 'Permanent Housing Only', value: 9
-    value 'PH', 'Permanent Housing', value: 10
-    value 'DAY_SHELTER', 'Day Shelter', value: 11
-    value 'PREVENTION', 'Homelessness Prevention', value: 12
-    value 'RRH', 'Rapid Re-Housing', value: 13
-    value 'CE', 'Coordinated Entry', value: 14
+    HudUtility.project_types.each do |id, description|
```
Comment (eanders) [inline drivers/hmis/app/graphql/types/hmis_schema/enums/project_type.rb:14]: Did you want HudUtility2024?

**Review by eanders — APPROVED** [review]
Looks good other than maybe the HudUtility call

File: drivers/hmis/app/graphql/types/hmis_schema/enums/project_type.rb (hunk 1 above)
Comment (gigxz) [reply drivers/hmis/app/graphql/types/hmis_schema/enums/project_type.rb:14]: not yet - doing that in  https://github.com/greenriver/hmis-warehouse/pull/3383

**Pull request merged**